typings/index.d.ts

```diff
@@ -9,6 +9,7 @@
 
 export type GeolocationConfiguration = {
   skipPermissionRequests: boolean;
+  authorizationLevel: "always" | "whenInUse"
```

Comment:
Same as with the flow types

```suggestion
  authorizationLevel: "always" | "whenInUse" | "auto"
```